src/state/lottery/index.ts

```diff
@@ -66,7 +66,7 @@ export const fetchUserTicketsAndLotteries = createAsyncThunk<
   const userTickets = await fetchTickets(account, currentLotteryId, totalTickets)
 
   // user has not bought tickets for the current lottery
-  if (userTickets.length === 0) {
+  if (!userTickets || userTickets.length === 0) {
```

Comment:
Add condition for when `fetchTickets` fails and returns null - so that it still returns `userLotteriesRes` and a users' history shows